.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Type of change
+
+- [ ] bug fix
+- [ ] new feature
+- [ ] documentation
+- [ ] other (describe)
+
+### Scope of changes
+
+Briefly describe the scope of changes. Include links to ShortCut or Jira ticket if applicable.
```

Comment:
Perhaps we should add a note here that our practice is to actually name our PR branches after the Shortcut stories they resolve and to include the `sc-XXXX` in the PR title , since we have automations set up to support workflow management in Shortcut